### DISCUSSION TITLE
Compatibility with Mac OSX Big Sur (M1 Chip)

### DISCUSSION THREAD

**Discussion by asing1234** [mtex-toolbox/mtex#762] [Ask Anything]
It seems that the current version of MTEX has compatible issue with the new M1 trip MacBook. Could anyone know if this issue could be solved, and how?

**Comment by lfgmorales** [comment]
did you go through this:


https://mtex-toolbox.github.io/installation


cheers



Luiz


Luiz F. G. Morales
ScopeM & Structural Geology and Tectonics Group
ETH Zürich
________________________________

**Reply by asing1234** [reply]
Thanks Luiz. I have followed the steps the article suggested, and it is working fine so far.

**Comment by kilir** [comment]
Hi,
having a c compiler suitable to build the Matlab interface for nfft  (and to actually make use of all the cores someone bought, it should support building with openMP) would be a good start. Does it?

On homebrew, it seems that gcc, libopm and other essential tools are available for Apple Silicon.

You can try to follow the compilation instructions Luiz provided already.

And also; does your Matlab version run natively or via Rosetta?

Cheers,
Rüdiger

**Reply by asing1234** [reply]
Hi Rüdiger, 

The instructions Luiz suggested made the MTEX working in my machine, so far. 
I have checked, it seems (not 100% sure though) the Matlab 2021 pre-release runs natively in my machine.

Thanks,
Kai
